docs/source/usage/index.rst

```diff
@@ -1,8 +1,50 @@
-============
-Use AiiDAlab
-============
+==========
+User guide
+==========
 
 
+***************
+Access AiiDAlab
+***************
+
+As a user, you have two options to access AiiDAlab:
+
+ 1. Log into one of the `open AiiDAlab servers <https://materialscloud.org/aiidalab>`_.
+ 2. Run the AiiDAlab docker container on your local machine.
```

Comment:
What about the option to just run the image from the docker hub?

Reply:
Thanks for the quick review!

> What about the option to just run the image from the docker hub?

That would fall under 2. ; I haven't yet adapted the instructions below to start with this but I'll do it now

Reply:
added - I'll go ahead with the merge